smtp/smtpc.c

```diff
@@ -140,6 +163,14 @@ smtpc_init(void)
 	load_func(libname, libcurl_handle, curl_easy_strerror);
 	load_func(libname, libcurl_handle, curl_slist_append);
 	load_func(libname, libcurl_handle, curl_slist_free_all);
+	load_func(libname, libcurl_handle, curl_version_info);
+
+	/* Verify that given libcurl supports smtp(s). */
+	curl_version_info_data *info = curl_version_info(7);
```

Comment:
"age should  be  set to  the  version  of this functionality by the time you write your program. This way, libcurl will always return a proper  struct  that  your program understands, while programs in the future might get a different struct." Cool idea.

Reply:
:)